scripts/add-test-user.sh

```diff
@@ -4,7 +4,7 @@ NETWORK=hint_nw
 HERE=$(realpath "$(dirname $0)")
 TEST_CONFIG=$HERE/../src/app/static/scripts/test.properties
 
-image=mrcide/hint-user-cli:master
+image=mrcide/hint-user-cli:mrc-3507
```

Comment:
This should be pulling from master but we do not have master image built yet for the new changes. I have created a ticket to revert back to master once this is merged. https://mrc-ide.myjetbrains.com/youtrack/issue/mrc-3510/Change-userCLI-image-target-to-the-one-built-for-master.